lightstep/opentelemetry-examples
# Add redis integration tests

What does this PR do?
---------------------
Integration tests for redis metrics

Why are you doing it?
---------------------
To see if it brings up redis successfully, then everything in logs exits in tf.

What are you asking the reviewer to do?
---------------------
If "review", leave off.

How can the reviewer test/see/play with the changes?
---------------------
If "run tests", leave off.

***
R/CC: _Who should know about this change?
@lightstep/otel-integrations
